lib/common/options.js

```diff
@@ -64,18 +64,16 @@ Options.from = function (_options) {
   // Populate modulesDir
   options.modulesDir = []
     .concat(options.modulesDir)
-    .concat(path.join(options.nuxtDir, 'node_modules'))
-    .filter(hasValue)
+    .concat(path.join(options.nuxtDir, 'node_modules')).filter(hasValue)
     .map(dir => path.resolve(options.rootDir, dir))
 
-  // Sanitize extensions
-  if (!options.extensions.includes('js')) {
-    options.extensions.unshift('js')
-  }
+  const extensionsToRemoveIfPresent = ['js', 'mjs']
```

Comment:
This name makes it sound like these extensions are being removed, whereas they are actually added, without duplication.

Reply:
Correct